pyvrp/cpp/Solution.h

```diff
@@ -95,27 +98,61 @@ class Solution
         [[nodiscard]] Load excessLoad() const;
 
         /**
-         * Total route duration, including waiting time.
+         * Total route duration, including travel, service and waiting time.
          */
         [[nodiscard]] Duration duration() const;
 
         /**
-         * Total duration of service on the route.
+         * Total duration of service on this route.
          */
         [[nodiscard]] Duration serviceDuration() const;
 
         /**
-         * Amount of time warp incurred along the route.
+         * Amount of time warp incurred on this route.
          */
         [[nodiscard]] Duration timeWarp() const;
 
+        /**
+         * Total duration of travel on this route.
+         */
+        [[nodiscard]] Duration travelDuration() const;
+
         /**
          * Total waiting duration on this route.
          */
         [[nodiscard]] Duration waitDuration() const;
 
         /**
-         * Release time of visits on this route.
+         * Start time of this route. This is the earliest possible time at which
+         * the route can leave the depot and have a minimal duration. It may be
+         * possible to leave earlier, but that would incur additional waiting
+         * time.
+         *
+         * .. note::
+         *
+         *    It may also be possible to leave later, without (additional)
+         *    violations of time window constraints or increasing the route
+         *    duration. In that case there is slack in the schedule: the amount
+         *    of slack can be obtained through :meth:`~slack`.
+         */
+        [[nodiscard]] Duration startTime() const;
+
+        /**
+         * End time of the route. This is equivalent to
+         * ``start_time + duration``.
+         */
+        [[nodiscard]] Duration endTime() const;
+
+        /**
+         * Time by which departure from the depot can be delayed without
+         * resulting in (additional) time window violations or increased route
```

Comment:
We interchangably use 'time window violations' (which sounds as 1 or 2 or 3 violations) and 'time warp' which is a related but slightly different concept. I don't really mind but wanted to raise the question if this does need explanation/reformulation?

Reply:
I've switched the two cases we have over to time warp.